src/main/java/frc/robot/subsystems/PTO/PTO.java

```diff
@@ -0,0 +1,87 @@
+package frc.robot.subsystems.PTO;
+
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Ports;
+
+/**
+ * This subsystem define the PTO.
+ * The PTO is responsible for switching between the climber and the shooter, because they are using the same motors.

```

Comment:
```suggestion
 * This subsystem defines the PTO.
 * The PTO is responsible for switching between the climber and the shooter because they are using the same motors.
```